### PR TITLE
Lid 1

### PR DESCRIPTION
**Pull request by zhukov-maxim**

### CODE REVIEW

File: src/components/LebenInDeutschland/LebenInDeutschland.test.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
-import App from './App';
+import App from './LebenInDeutschland';
 
 test('renders learn react link', () => {
   render(<App />);
```
Comment (yuliya-zhukova) [inline src/components/LebenInDeutschland/LebenInDeutschland.test.tsx]: image

File: .stylelintrc [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "extends": [
+    "stylelint-config-standard",
+    "stylelint-config-prettier"
+  ],
+  "rules": {
+    "selector-pseudo-element-colon-notation": "double"
+  }
+}
```
Comment (yuliya-zhukova) [inline .stylelintrc:9]: image

Comment (zhukov-maxim) [reply .stylelintrc:9]: Let's improve Stylelint/Prettier configuration in a separate task.
https://mzhukov.atlassian.net/browse/LID-6

File: src/components/LebenInDeutschland/LebenInDeutschland.test.tsx (hunk 1 above)
Comment (zhukov-maxim) [reply src/components/LebenInDeutschland/LebenInDeutschland.test.tsx]: Thanks. Fixed.

**Pull request merged**